Sources/TuistLoader/Up/Carthage.swift

```diff
@@ -96,29 +96,29 @@ final class Carthage: Carthaging {
         }
 
         var outdated: [String] = []
-        let carfileResolved = try FileHandler.shared.readTextFile(cartfileResolvedPath)
```

Comment:
Do we still need the Carthage up command? What do you think if we remove it for 2.0?

Reply:
Not super familiar with carthage, let's see what @laxmorek thinks

Reply:
if we support Carthage within `Dependencies.swift`, as a user I would also expect it supported

Reply:
I agree that we should remove it. I wrote about it a few months ago in my comment [here](https://github.com/tuist/tuist/issues/2981#issuecomment-850526334).

I have seen that @pepibumur has already removed it in this [PR](https://github.com/tuist/tuist/pull/3443)